src/pages/policies/policies.tsx

```diff
@@ -65,24 +65,51 @@ export default function Policies() {
     );
   }
 
-  return (
-    <PoliciesLayout>
-      {filteredPolicies.length > 0 ? (
-        <>
-          <div className={styles.resultsInfo}>
-            {filteredPolicies.length} result(s)
-            {query && <> for "{query}"</>}
-          </div>
-          <PolicyList policies={pagedPolicies} />
-          <Pagination totalResults={filteredPolicies.length} currentPage={currentPage} className={styles.pagination} />
-        </>
-      ) : policies.length === 0 ? (
-        <p className={styles.emptyState}>There are no policies linked to your account.</p>
-      ) : (
+  if (!filteredPolicies.length) {
+    let policyQualifier;
+    if (filter === 'active') {
+      policyQualifier = <span className={styles.highlight}>active </span>;
```

Comment:
I'd prefer a CSS style instead of extra space.

Reply:
This case is a bit different as it's not a block element, but a piece of inline text that gets highlighted, so adding the space will allow us to assert the text content of the parent element like "We couldn't find any active policies with ..." vs "We couldn't find any activepolicies with ..."

Reply:
I see.